doc/news/changes.h

```diff
@@ -140,7 +140,30 @@ inconvenience this causes.
   - TrilinosWrappers::*Vector*::compress with an Epetra_CombineMode
     argument
   <br>
-  (Wolfgang Bangerth, 2014/12/29-2015/01/09)
+  This release also removes the deprecated class MGDoFHandler. The
+  functionality of this class had previously been incorporated into
+  the DoFHandler class. Unlike the changes above, if you were still
+  using this class, you will need to do the following changes to
+  your code:
+  - Where you called <code>mg_dof_handler.distribute_dofs()</code>
+    you now also need to explicitly call
+    <code>mg_dof_handler.distribute_mg_dofs()</code>.
+  - If you called <code>mg_dof_handler.begin(level)</code>, you
+    will now have to write this as
+    <code>mg_dof_handler.begin_mg(level)</code> to make clear that
```

Comment:
Okay, I forgot that we already adjusted MeshWorker to take advantage of level iterators. Thanks for clarifying.